app/components/point.jsx

```diff
@@ -2,6 +2,7 @@
 // https://github.com/EnCiv/civil-pursuit/issues/76
```

Comment:
since there are no changes to this file, please revert it to origin/master:
git checkout origin/master app/components/point.jsx
git add app/components/point.jsx